src/workspace/resolver.rs

```diff
@@ -543,30 +543,6 @@ impl WorkspaceResolver {
     };
 
     // 2. Try to resolve the bare specifier to a workspace member
-    if resolve_error.is_unmapped_bare_specifier() {
-      for member in &self.jsr_pkgs {
-        if let Some(path) = specifier.strip_prefix(&member.name) {
-          if path.is_empty() || path.starts_with('/') {
-            let path = path.strip_prefix('/').unwrap_or(path);
-            return self.resolve_workspace_jsr_pkg(
-              member,
-              JsrPackageReqReference::from_str(&format!(
-                "jsr:{}{}/{}",
-                member.name,
-                member
-                  .version
-                  .as_ref()
-                  .map(|v| format!("@^{}", v))
-                  .unwrap_or_else(String::new),
-                path
-              ))
-              .unwrap(),
-            );
-          }
-        }
-      }
-    }
```

Comment:
One scenario that bare specifiers are nice in is when you don't publish the workspace members to jsr. It feels weird referring to them with the jsr: prefix in that case.